tests/acceptance/usage-in-acceptance-test.js

```diff
@@ -0,0 +1,33 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import { visit, click, fillIn, waitUntil } from 'ember-native-dom-helpers/test-support/helpers';
```

Comment:
Yes, that should be documented, although my intention with this PR is bring attention over how this syntax and approach feels. 
@machty p.e suggested making `find`/`findAll` async, and accept modifiers like `find('.some-class', { count: 5 })`. I then suggested the existence of the `waitUntil` helper and keep finders sync. That kind of feedback.